code/src/reducers/todoList.js

```diff
@@ -0,0 +1,56 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+  list: [
+    {
+      id: 1,
+      task: 'Work with todo project',
+      isCompleted: false,
+    },
+    {
+      id: 2,
+      task: 'Cook dinner',
+      isCompleted: false,
+    },
+    {
+      id: 3,
+      task: 'Do laundry',
+      isCompleted: false,
+    }
+  ]
+}
+
+export const todoList = createSlice({
+  name: 'todoList',
+  initialState,
+  reducers: {
+    addTask: (state, action) => {
+      const id = Math.max(...state.list.map(item => item.id)) + 1
```

Comment:
Tried to understand what the id is that you get from this line of code, but not really following :)

Reply:
these codes is to create a new Id which is increasing from the current one.